package.json

```diff
@@ -74,6 +74,12 @@
     "@babel/preset-env": "^7.11.5",
     "@babel/preset-react": "^7.10.4",
     "@babel/register": "^7.11.5",
+    "@liatrio/semantic-release-helm": "^2.0.0",
+    "@semantic-release/commit-analyzer": "^9.0.2",
+    "@semantic-release/git": "^10.0.1",
+    "@semantic-release/github": "^8.0.2",
+    "@semantic-release/npm": "^9.0.1",
+    "@semantic-release/release-notes-generator": "^10.0.3",
```

Comment:
we don't need all of these - most of these are already bundled with the `semantic-release` dependency.  refer to the plugins that are used within the sparky repo: https://github.com/liatrio/sparky/blob/main/package.json#L23-L25

Reply:
yea we were confused about that. we assumed we wanted all of them but noticed that sparky only had a few. sorry about that

Reply:
no need to apologize, here's the list of default plugins if you were curious: https://github.com/semantic-release/semantic-release/blob/master/docs/usage/plugins.md#default-plugins